### PR TITLE
Add 'Zeebe Modeler.app' v0.1.0

### PR DESCRIPTION
**Pull request by jangalinski**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, if **updating a cask**:

- [ ] `sha256` changed but `version` stayed the same ([what is this?][version-checksum]).  
      I’m providing public confirmation below.

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not already refused in [closed issues].
- [ ] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed
[the correct repo]: https://github.com/caskroom/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

**Comment by commitay** [comment]
Thank you for your PR!

> [Rejected Casks](https://github.com/caskroom/homebrew-cask/blob/master/doc/faq/rejected_casks.md#rejected-casks)
> The Cask is too obscure (for example, a self-submitted app from a GitHub repository that is not notable enough (< 10 stars)).

I'd suggest [creating your own tap.](https://github.com/Homebrew/brew/blob/master/docs/How-to-Create-and-Maintain-a-Tap.md)

**Comment by jangalinski** [comment]
Hi ... I'd like to clarify:  this i the graphical modeler for the zeebe-io microservice orchestration tool, you find it here: https://github.com/zeebe-io/zeebe
we just keep it in a separate repo. zeebe is the "next big thing" of camunda (see camunda.org or https://github.com/caskroom/homebrew-cask/blob/master/Casks/camunda-modeler.rb) for reference. We want to start early with cask this time, and as you notice, meanwhile this got mor than 10 stars. Could you reconsider?

File: Casks/zeebe-modeler.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'zeebe-modeler' do
+  version '0.1.0'
+  sha256 '2d6d84667bd01e46c6df45a182b7cfdd746a6c3475217087524f7062f9aef017'
+
+  # github.com/zeebe-io/zeebe-modeler was verified as official when first introduced to the cask
+  url "https://github.com/zeebe-io/zeebe-modeler/releases/download/#{version}/zeebe-modeler-darwin-x64.tar.gz"
+  name 'Zeebe Modeler'
```
Comment (commitay) [inline Casks/zeebe-modeler.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)

**Review by commitay — CHANGES_REQUESTED** [review]

**Comment by commitay** [comment]
> Hi ... I'd like to clarify: this i the graphical modeler for the zeebe-io microservice orchestration tool, you find it here: https://github.com/zeebe-io/zeebe
we just keep it in a separate repo. zeebe is the "next big thing" of camunda (see camunda.org or https://github.com/caskroom/homebrew-cask/blob/master/Casks/camunda-modeler.rb) for reference. We want to start early with cask this time, and as you notice, meanwhile this got mor than 10 stars. Could you reconsider?

Only reopening because of the increased interest.

**Comment by jangalinski** [comment]
thanks!

**Pull request merged**